iosApp/Survey/Sources/Presentation/Coordinator/RouteCoordinator.swift

```diff
@@ -71,6 +71,7 @@ extension RouteCoordinator: SurveySelectionCoordinator {
 extension RouteCoordinator: SurveyDetailCoordinator {
 
     func backToHome() {
+        guard routes.count > 1 else { return }
```

Comment:
Optional: It would be better for us to have a simple explanation comment on why there is a need for this guard check `routes.count > 1` before dismissing. 🙏